HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -110,6 +109,193 @@ def test_ducts
     _test_default_duct_values(hpxml, expected_supply_locations, expected_return_locations, expected_supply_areas, expected_return_areas, expected_n_return_registers)
   end
 
+  def test_pv
+    hpxml_name = 'base-pv.xml'
+    hpxml = HPXML.new(hpxml_path: File.join(@root_path, 'workflow', 'sample_files', hpxml_name))
+    XMLHelper.write_file(hpxml.to_oga, @tmp_hpxml_path)
+    model, hpxml = _test_measure(@args_hash)
+    expected_interver_efficiency = [0.96, 0.96]
+    expected_system_loss_frac = [0.14, 0.14]
```

Comment:
Since these values are the same as the default, it's possible the values are being overridden with defaults and we wouldn't know. How about changing their values in the hpxml object (i.e., after you read in the HPXML file) and then making sure those values are reflected in the expected values?